tasks/system_probe.py

```diff
@@ -1370,3 +1370,22 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
         output_path = image.translate(str.maketrans('', '', string.punctuation))
         ctx.run(f"docker pull --platform linux/{arch} {image}")
         ctx.run(f"docker save {image} > {os.path.join(output_dir, output_path)}.tar")
+
+
+@task
+def test_microvms(
+    ctx, security_groups=None, subnets=None, instance_type_x86=None, instance_type_arm=None, destroy=False
+):
+    args = []
+    if security_groups is not None:
+        args.append(f"--sgs {security_groups}")
+    if subnets is not None:
+        args.append(f"--subnets {subnets}")
+    if instance_type_x86 is not None:
+        args.append(f"--instance-type-x86 {instance_type_x86}")
+    if instance_type_arm is not None:
+        args.append(f"--instance-type-arm {instance_type_arm}")
+    if destroy:
+        args.append("--destroy")
+
+    ctx.run(f"cd ./test/new-e2e && go run ./scenarios/system-probe/main.go --name usama-saqib-test {' '.join(args)}")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have you considered using `go test` instead ? Might be early, we are defining our test api this week, but you could eventually create a stack using `testify` like in unit tests